docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -416,38 +458,72 @@ When supporting multiple layouts, define the multiple layout nodes and then set
-:::
+### Chosen Node
 
-## Default Keymap
+Set the `chosen` node to a defined "default" physical layout.
 
-Each keyboard should provide a default keymap to be used when building the firmware, which can be overridden and customized by user configs. For "shield keyboards", this should be placed in the `boards/shields/<shield_name>/<shield_name>.keymap` file. The keymap is configured as an additional devicetree overlay that includes the following:
+```dts
+/ {
+    chosen {
+        zmk,physical-layout = &default_layout;
+        // Other chosen items
+    };
+};
+```
 
-- A node with `compatible = "zmk,keymap"` where each child node is a layer with a `bindings` array that binds each key position to a given behavior (e.g. key press, momentary layer, etc).
+If you define multiple physical layouts, users can select a different layout by overriding the `zmk,physical-layout` chosen node in their keymap file.
 
-Here is an example simple keymap for the Kyria, with only one layer:
+:::note
+If all of your physical layouts use the same `kscan` node under the hood, you can skip setting the `kscan` property on each layout and instead assign the `zmk,kscan` chosen node to your single kscan instance:
 
-<KeymapExampleFile />
+```dts
+/ {
+    chosen {
+        zmk,kscan = &kscan0;
+        zmk,physical-layout = &default_layout;
+        // Other chosen items
+    };
+};
+```
 
-:::note
-The two `#include` lines at the top of the keymap are required in order to bring in the default set of behaviors to make them available to bind, and to import a set of defines for the key codes, so keymaps can use parameters like `N2` or `K` instead of the raw keycode numeric values.
 :::
 
-### Keymap Behaviors
+## Default Keymap
+
+Each keyboard should provide a default keymap to be used when building the firmware, which can be overridden and customized by user configs.
+For "shield keyboards", this should be placed in the `boards/shields/my_keyboard/my_keyboard.keymap` file.
+The keymap is configured as an additional devicetree overlay that includes the following:
+
+Here is an example simple keymap for a 3x3 macropad, with only one layer:
+
+```dts title="my_keyboard.keymap"
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer { // Layer 0
+            // -------------------------------------
+            // |     Z     |     M     |     K     |
+            // |     A     |     B     |     C     |
+            // |     D     |     E     |     F     |
+            bindings = <
+                &kp Z    &kp M    &kp K
+                &kp A    &kp B    &kp C
+                &kp D    &kp E    &kp F
+            >;
+        };
+    };
+ };
+```
 
-For documentation on the available behaviors for use in keymaps, see the [overview page for behaviors](../../keymaps/behaviors/index.mdx).
+See [Keymaps](../../keymaps/index.mdx) for information on defining keymaps in ZMK.
 
 ## Metadata
```

Comment:
I wonder if this should be taken out and replaced with a pointer as well, since the same example exists in the dedicated page and this section doesn't add much.